### PR TITLE
Another batch of Debian packaging cleanups

### PR DESCRIPTION
**Pull request by daissi**
Let's start with the serious things.
Now, packages build correctly in a clean sid pbuilder env.

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -36,7 +36,8 @@ else
 endif
 
 override_dh_auto_build:
-	dh_auto_build -- lp_all
+	echo "No need to build DKMS modules at packages build time."
+	echo "DKMS modules are built at packages installation time."
```
Comment (z3ntu) [inline debian/rules]: This was actually done on purpose so launchpad tests, if the kernel modules can be built. But now we have CI so it's not really important anymore.

**Comment by z3ntu** [comment]
Except 9eff02909e0865ab47062f729f6d444a450bd65d the changes look good. So how should people know that they have to add their user to the plugdev group?

**Comment by daissi** [comment]
postinst is not the best way to tell user to add themself to plugdev group because it can fail in many cases:
- Users may not look the shell during packages installation, so they can skip the message about the plugdev group.
- Or, user which will install openrazer is not obliviously the same user that the one which will use openrazer, so those users may not be aware.
- Or, if someone create a new user on its system after installing openrazer packages, he will not be aware about the plugdev group.
- etc

Maybe you can switch from plugdev group to uaccess tag in the udev file. uaccess tag seems to replace the usage of plugdev group on all system with systemd (random examples: [Debian](https://wiki.debian.org/USB/GadgetSetup), [RedHat](https://bugzilla.redhat.com/show_bug.cgi?id=1270102), [Arch Linux](https://wiki.archlinux.org/index.php/Talk:Udev#Use_of_.27uaccess.27_instead_of_GROUP_and_MODE.3F), [systemd](https://github.com/systemd/systemd/issues/4288)). So, this problem could disappear by using uaccess tag.

Another solution could be to test at the launch time of an end user software (Polychromatic, RazerGenie, etc) if the user is in the group and if not, it warn them.

**Comment by z3ntu** [comment]
uaccess doesn't work for sysfs files. Front-end check is possible but until now the postinstall script seems to have worked most of the times so I'd rather keep it.

File: debian/openrazer-kernel-modules-dkms.postinst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+#DEBHELPER#
+
+# Actually needs -e but if added, it gets printed out literally and it works without just fine.
```
Comment (z3ntu) [inline debian/openrazer-kernel-modules-dkms.postinst]: that comment can be removed then

**Review by z3ntu — APPROVED** [review]
LGTM

**Pull request merged**